src/ripple/app/main/Application.cpp

```diff
@@ -614,9 +615,10 @@ class ApplicationImp
         return m_acceptedLedgerCache;
     }
 
-    void gotTXSet (uint256 const& setHash, std::shared_ptr<SHAMap> const& set)
+    void gotTXSet (std::shared_ptr<SHAMap> const& set, bool fromAcquire)
     {
-        m_networkOPs->mapComplete (setHash, set);
+        if (set)
+            m_networkOPs->mapComplete (set, fromAcquire);
```

Comment:
This should be documented in the API, preferably in the javadoc explaining the arguments of the callback.